src/java/org/apache/cassandra/index/sai/plan/QueryController.java

```diff
@@ -556,6 +570,18 @@ private List<CloseableIterator<ScoredPrimaryKey>> orderSstables(QueryViewBuilder
         return results;
     }
 
+    /**
+     * Computes the ANN search limit to be used for searching in the sstable index.
+     * @param limit the desired number of TopK rows
+     * @param totalRows total number of rows in all sstables
+     */
+    private int sstableLimit(SSTableReader sstable, int limit, long totalRows)
+    {
+        return V3OnDiskFormat.REDUCE_TOPK_ACROSS_SSTABLES
```

Comment:
note for the future: it looks like a code smell to have references to V3OnDiskFormat in this class

Reply:
It's a feature flag that we hope to inline to true in the next release